### ISSUE TITLE
What is Jamstack?

### ISSUE THREAD

**Issue by Tschrock** [jamstack/jamstack.org#755]
I just heard the term and am trying to figure out what it means, and the website is not really helping. The "What is Jamstack?" page says it's an architecture? I would assume since it's a *stack that it's an acronym for some kind software stack, but the site doesn't mention an acronym and it doesn't seem to be a concrete software stack like MEAN or LAMP. There are a few parts that kind of make it sound like a service or a devtool, but if I'm right it's actually it's more of an ideology? The homepage and "what is" page talk a lot about the benefits of Jamstack and some of the principles, but I can't find anywhere that describes what it actually *is*. Like, concretely, what it means, not just a sales pitch. (Consider this as both a question and a "your website is kinda confusing" bug report.)

**Comment by talves** [comment]
There have been some great discussions at the discord server for [Jamstack here](https://discord.gg/cdwaMkAuhX).

[Salma](https://twitter.com/whitep4nth3r) , one of the moderators, [has a post just released](https://whitep4nth3r.com/blog/what-is-jamstack/) about this question.

**Comment by ubyjvovk** [comment]
Yeah, I'm an experienced developer and the front page copy makes no sense to me whatsoever. After spending a few minutes on the website, I still have no idea what JAMstack does. Why is it a stack, what are the components? Is it a static site generator? Is it an API wrapper? ¯\_(ツ)_/¯  


"improving flexibility, scalability, performance, and maintainability." is MBA-speak and can mean *anything at all*. Nginx does all of these things. So does a Rust compiler. So does my Macbook. And the whole AWS ecosystem does it, too.

Seriously, the front copy could use some improvement.

**Comment by saurabhsingh72487-hub** [comment]
discuss more about on the jamstack please
